contents/tutorials/ios-ab-tests.md

```diff
@@ -0,0 +1,194 @@
+
+## Creating a new iOS app
+
+Our app will have two screens: The first screen will have a button which will take you to a second screen. The second screen will have it's background color set to `red` or `green`, depending on if the user is in the `control` or `test` group. Finally, the second screen will have a button which captures an event when it's pressed. We'll use this event as our goal metric for our test.
+
+The first step is to create a new app. Open XCode and click "Create new project". Select iOS as your platform, then "App" and press next. Give your app a name, select `SwiftUI` as the interface, and the defaults for everything else. Click next and then "Create".
+
+Then, replace your code in `ContentView` with the following:
+
+```swift
+// in ContentView.swift
+import SwiftUI
+
+struct ContentView: View {
+    @State private var navigateToFeatureScreen = false
+
+    var body: some View {
+        NavigationStack {
+            VStack {
+                Image(systemName: "globe")
+                    .imageScale(.large)
+                    .foregroundStyle(.tint)
+                Text("Hello, world!")
+
+                Button("Go to Next Screen") {
+                    navigateToFeatureScreen = true
+                }
+                .padding()
+                .background(Color.blue)
+                .foregroundColor(.white)
+                .cornerRadius(10)
+            }
+            .padding()
+            .navigationDestination(isPresented: $navigateToFeatureScreen) {
+                FeatureScreenView(isTestVariant: false) // We'll update this later
+            }
+        }
+    }
+}
+```
+
+Then, create a new SwiftUI View called `FeatureScreenView`. Replace the default code in the new file with the following:
+
+```swift
+// in FeatureScreenView.swift
+import SwiftUI
+
+struct FeatureScreenView: View {
+    var isTestVariant: Bool
+
+    var body: some View {
+        ZStack {
+            Color(isTestVariant ? .green : .red)
+                .edgesIgnoringSafeArea(.all)
+
+            Button("Click Me!") {
+            }
+            .padding()
+            .background(Color.blue)
+            .foregroundColor(.white)
+            .cornerRadius(10)
+        }
+    }
+}
+```
+
+Our basic set up is now complete. Build and run your app to test that it's working.
+
+![Basic setup of the iOS app](../images/tutorials/ios-ab-tests/app-setup.mp4)
+
+## Adding PostHog to your iOS app
+
+First, add [posthog-ios](https://github.com/PostHog/posthog-ios) as a dependency to your app using [Swift Package Manager](https://developer.apple.com/documentation/xcode/adding-package-dependencies-to-your-app) (or if you prefer, you can use [CocoaPods](/docs/libraries/ios#cocoapods) or [Carthage](/docs/libraries/ios#carthage)). To add the package dependency to your Xcode project, select File > Add Package Dependency and enter the URL `https://github.com/PostHog/posthog-ios.git`. Select `posthog-ios` and click Add Package.
```

Comment:
Shouldn't we link the https://posthog.com/docs/libraries/ios docs here instead of github?

Reply:
I think since this is a dependency that you're going to add to Swift Package manager, it's better to link to the github